callback_plugins/ansible_version_check.py

```diff
@@ -0,0 +1,33 @@
+# -*- encoding:utf-8 -*-
+# https://adamj.eu/tech/2016/08/19/restricting-the-ansible-version-in-use/
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import sys
+
+import ansible
+
+try:
+    # Version 2.0+
+    from ansible.plugins.callback import CallbackBase
+except ImportError:
+    CallbackBase = object
+
+
+def print_red_bold(text):
```

Comment:
Set the message string and then use the built-in `_display` (it's a part of callback objects). Here's an example where they detect a task failed and then use the `C` constants to print with the correct red color:

https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/callback/default.py#L55-L57

You'll have to make sure you initialize the parent class though: https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/callback/default.py#L41

to make sure that the `_display` gets set: https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/callback/__init__.py#L55

Or add the kwarg yourself to the ``__init__`` definition (I think).